cmd/gen.go

```diff
@@ -45,8 +47,10 @@ var genCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(genCmd)
 
-	genCmd.Flags().StringVarP(&issuerPrivateKey, "issuer-private-key", "i", "", "Path to PEM encoded Ed25519 private key of delegation issuer")
-	Must(genCmd.MarkFlagRequired("issuer-private-key"))
+	genCmd.Flags().StringVarP(&issuerPrivateKeyFile, "issuer-private-key-file", "f", "", "Path to PEM encoded Ed25519 private key of delegation issuer")
+	genCmd.Flags().StringVarP(&issuerPrivateKey, "issuer-private-key", "i", "", "Multibase encoded Ed25519 private key of delegation issuer")
+	genCmd.MarkFlagsMutuallyExclusive("issuer-private-key-file", "issuer-private-key")
+	genCmd.MarkFlagsOneRequired("issuer-private-key-file", "issuer-private-key")
```

Comment:
Could we instead leave the behavior of `issuer-private-key` unchanged and call the new flag `issuer-private-key-raw` or something?
Alternatively, we'll need to update the docs to reflect this change.

Reply:
oh, good point, let me update the docs